flake.nix

```diff
@@ -6,7 +6,7 @@
     flake-parts = { url = "github:hercules-ci/flake-parts"; inputs.nixpkgs-lib.follows = "nixpkgs"; };
     hercules-ci-effects = { url = "github:hercules-ci/hercules-ci-effects"; inputs.nixpkgs.follows = "nixpkgs"; };
     flake-compat = { url = "github:edolstra/flake-compat"; flake = false; };
-    neovim-flake = { url = "github:neovim/neovim?dir=contrib"; inputs.nixpkgs.follows = "nixpkgs"; };
+    neovim-flake = { url = "path:./neovim"; inputs.nixpkgs.follows = "nixpkgs"; };
```

Comment:
I get a `cannot fetch path because it uses a relative path. I believe we should just keep this flake and merge neovim's one into it (whcih could also import *.nix files)

Reply:
Man I tested this when I put it up and swear I didn't get this error :laughing: Thanks for pointing this out, I will merge the two :+1: